### PR TITLE
Implement SkillSlideshow Servlet #1293

### LINKED ISSUE #1293: Implement SkillSlideshow Servlet
**Actual Behaviour**

Currently not present. PR: https://github.com/fossasia/susi.ai/issues/2455 requires it

**Expected Behaviour**

Implement services for saving and fetching data from the database

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by fragm3**
Fixes #1293

Changes: The slider images are currently hardcoded into the frontend. Implement services for saving and fetching slider data from the backend

Needed for PR: https://github.com/fossasia/susi.ai/issues/2455

For fetching slider image: http://localhost:4000/cms/getImage.png?sliderImage=Android-Slider.jpg

### CODE REVIEW

**Comment by fragm3** [comment]
@akshatnitd @Orbiter please review

File: src/ai/susi/server/api/cms/SkillSlideshowService.java [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+import ai.susi.json.JsonTray;
+import ai.susi.server.*;
+import ai.susi.server.Authorization;
+import org.json.JSONObject;
+
+import javax.servlet.http.HttpServletResponse;
+
+/**
+ * This Servlet gives a API Endpoint to add, modify and delete different API
+ * keys used by SUSI. It requires user role to be ADMIN or above ADMIN example:
```
Comment (akshatnitd) [inline src/ai/susi/server/api/cms/SkillSlideshowService.java]: Change it.

File: src/ai/susi/server/api/cms/SkillSlideshowService.java [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+    }
+
+    @Override
+    public ServiceResponse serviceImpl(Query call, HttpServletResponse response, Authorization authorization,
+            final JsonObjectWithDefault permissions) throws APIException {
+        if (call.get("pathname", null) == null || call.get("image_name", null) == null) {
+            throw new APIException(400, "Bad Request. No enough parameter present");
+        }
+
+        String pathname = call.get("pathname", null);
```
Comment (akshatnitd) [inline src/ai/susi/server/api/cms/SkillSlideshowService.java]: Pathname should not be entered by user.

Comment (fragm3) [reply src/ai/susi/server/api/cms/SkillSlideshowService.java]: Pathname is redirect link, which rename it to redirect_link, to avoid confusion

**Comment by fragm3** [comment]
@akshatnitd made the changes, please review again

**Comment by fragm3** [comment]
@akshatnitd made the changes.

**Pull request merged**